blocks/product-list-page/product-list-page.js

```diff
@@ -42,7 +42,7 @@ export default async function decorate(block) {
       },
     },
     context: {
-      customerGroup: await getConfigValue('commerce-customer-group'),
+      customerGroup: await getConfigValue('commerce.queryparam.cs.Magento-Customer-Group'),
```

Comment:
If we dedupe the config, so that we only have values in one place when possible, then we will end up with quite a lot of these sort of calls, where the key doesn't really make sense in the context of what we're passing the value to.

So it's a choice, I guess. Either we keep multiple duplicated values within the config, so the keys are more explicit, or we remove extra values and rely on just a single config value.

Reply:
Ideally the PLP and LS widget are updated to allow for passing of headers in a single `headers` object rather than as top-level props on this "config" object. Then we could use the same iteration logic as the PDP dropin in this PR.